.github/workflows/sync-bioconductor.yml

```
@@ -0,0 +1,54 @@
+      uses: actions/checkout@v4
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        fetch-depth: 0  # Need full history for rebase
+        
+    - name: Setup SSH key
+      uses: webfactory/ssh-agent@v0.9.0
+      with:
+        ssh-private-key: ${{ secrets.BIOC_SSH_PRIVATE_KEY }}
+        
+    - name: Add SSH host key
+        run: |
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 21: mapping values are not allowed in this context

(syntax-check)